code/__DEFINES/colors.dm

```diff
@@ -113,6 +113,8 @@
 #define COLOR_THEME_GLASS "#75A4C4"
 #define COLOR_THEME_CLOCKWORK "#CFBA47"
 
+///Colors for eigenstates
+#define COLOR_EIGENSTATE "#9999FF"
```

Comment:
This should be a reference to another color, such as COLOR_PERIWINKLE, in this case. Ideally, other stuff might want to use this color.

Reply:
Changed!